shanoir-ng-front/src/app/datasets/dataset/dataset.component.html

```diff
@@ -59,6 +59,7 @@ <h2 class="header command-zone"i18n="Create dataset|Title@@datasetDetailCreateTi
 				<button *ngIf="mode=='view' && hasDownloadRight && dataset.type != 'Eeg'" [disabled]="downloading" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('dcm')">Download Dicom<i class="fas fa-download"></i></button>
 				<button *ngIf="mode=='view' && hasDownloadRight && dataset.type != 'Eeg'" [disabled]="downloading" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('nii')">Download Nifti<i class="fas fa-download"></i></button>
 				<button *ngIf="mode=='view' && hasDownloadRight && dataset.type == 'Eeg'" [disabled]="downloading" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('eeg')">Download EEG<i class="fas fa-download"></i></button>
+				<button *ngIf="mode=='view' && hasDownloadRight" type="button" class="right-icon" (click)="seeDicomMetadata()">Show DICOM metadata<i class="fas fa-file-code"></i></button>
```

Comment:
This should not appear on EEG data I guess

Reply:
I added a check